tests/stdlib/tstrutils.nim

```diff
@@ -28,6 +28,26 @@ template main() =
     doAssert strip("sfoofoofoos", leading = false, chars = {'s'}) == "sfoofoofoo"
     doAssert strip("sfoofoofoos", trailing = false, chars = {'s'}) == "foofoofoos"
 
+    block:
+      var a = "xxxxxx"
```

Comment:
* s/var/let/

* use 1 liners instead for all tests you added, eg:
`doAssert "".strip(chars={'x'}).len == 0`

but feel free to add 1 test which uses an auxiliary variable